swift/ql/src/change-notes/2024-08-01-swift-linux.md

```diff
@@ -0,0 +1,5 @@
+---
+category: deprecated
+---
+
+* Swift analysis on Ubuntu runner images is no longer supported. Please migrate to a macOS runner if this affects you.
```

Comment:
This looks like a very odd place to add this change note, as this directory is meant for change notes that concern changes to Swift queries.

The CLI change notes might be a more appropriate place.

Reply:
Good point. See here instead: https://github.com/github/semmle-code/pull/50804